projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.ts

```diff
@@ -19,11 +22,11 @@ export class CartItemComponent implements OnInit {
   @Input()
   item: Item;
   @Input()
-  potentialProductPromotions: any[];
```

Comment:
Removing//changing an input of existing component is considered a breaking cahnge.